src/store/collectibles/reducers.ts

```diff
@@ -16,7 +17,8 @@ export function collectibles(state: CollectiblesState = initialCollectibles, act
             action.payload.collectibles.forEach(collectible => {
                 allCollectibles[collectible.tokenId] = collectible;
             });
-            return { ...state, allCollectibles };
+            const allCollectiblesFetchStatus = AllCollectiblesFetchStatus.Success;
+            return { ...state, allCollectibles, allCollectiblesFetchStatus };
         case getType(actions.selectCollectible):
```

Comment:
`@TODO`: handle the error action type, which also implies to handle that on the UI. For a starter/base UI project I guess this is quite ok.